app/views/pets/edit.html.erb

```diff
@@ -1,14 +1,14 @@
 <h1>Update Pet</h1>
 <%= form_tag "/pets/#{@pet.id}", method: :patch  do %>
-  <%= label_tag :name %>
-  <%= text_field_tag :name, "#{@pet.name}" %> <br>
-  <%= label_tag :image_path %>
-  <%= text_field_tag :image_path, "#{@pet.image_path}" %><br>
-  <%= label_tag :description %>
-  <%= text_field_tag :description, "#{@pet.description}" %><br>
-  <%= label_tag :approximate_age %>
-  <%= text_field_tag :approximate_age, "#{@pet.approximate_age}" %><br>
-  <%= label_tag :sex %>
-  <%= text_field_tag :sex, "#{@pet.sex}" %><br>
+  <%= label_tag "Name: "%>
+  <%= text_field_tag :name, "#{@pet.name}" %> <br><br>
+  <%= label_tag "Image Path: " %>
+  <%= text_field_tag :image_path, "#{@pet.image_path}" %><br><br>
+  <%= label_tag "Description: " %>
+  <%= text_field_tag :description, "#{@pet.description}" %><br><br>
+  <%= label_tag "Approximate Age: " %>
+  <%= text_field_tag :approximate_age, "#{@pet.approximate_age}" %><br><br>
```

Comment:
I could have made a class for each one of these and styled them in the CSS file instead of putting `<br>` after every line.